setup.py

```diff
@@ -16,7 +16,7 @@
 
 setup(
     name="shimmingtoolbox",
-    python_requires=">=3.7",
+    python_requires=">=3.6",
```

Comment:
I changed this to support Ubuntu Xenial (which is one of the machines I'm on). It's not strictly necessary; relying on 3.7+ has some benefits afterall.